src/assets/scripts/client/AppController.js

```diff
@@ -299,6 +302,21 @@ export default class AppController {
         this.updateViewControls();
     }
 
+    /**
+     * event callback fired from within the `TrafficRateController` when traffic should be cleared and respawned.
+     *
+     * @for AppController
+     * @method onTrafficReset
+     */
+    onTrafficReset() {
+        EventTracker.recordEvent(TRACKABLE_EVENT.AIRPORTS, 'traffic-reset', AirportController.current.icao);
+        this.aircraftController.aircraft_remove_all();
+        this.scopeModel.radarTargetCollection.reset();
+        AirportController.current.departureRunwayModel.reset();
```

Comment:
If this is being done to clear the departure queue, you should probably do it for all runways, since users can assign any runway they choose.